### ISSUE TITLE
Hires prompt takes only first prompt of batch

### ISSUE THREAD

**Issue by PixelClassicist** [lllyasviel/stable-diffusion-webui-forge#1330]
As mentioned earlier, when using wildcards, I observed that after the batch was completed, the wildcards were correctly applied and swapped within the normal prompt. However, the hires prompt only utilized the very first prompt from the batch throughout the entire process.

**Comment by Urotr0pine** [comment]
Yeah, same. Wildcards works well within one batch with different butchsize, but if there is more then one butch - all subsequent butches have same hires prompt as first butch

**Comment by PixelClassicist** [comment]
Would reallylike an update to this, this issue still persists in the latest version

**Comment by MarkWar88** [comment]
any workarround or fix?
